lib/protocol/message_xdr.go

```diff
@@ -700,6 +712,20 @@ Folder Structure:
 \                     ID (variable length)                      \
 /                                                               /
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                    Read Only (V=0 or 1)                     |V|
```

Comment:
This is rather ugly from a wire protocol point of view. Can we not just define these in Flags?